distributed/scheduler.py

```diff
@@ -5516,7 +5520,7 @@ def handle_task_erred(self, key=None, **msg):
         recommendations: dict
         client_msgs: dict
         worker_msgs: dict
-        r: tuple = self.stimulus_task_erred(key=key, **msg)
+        r: tuple = self.stimulus_task_erred(key=key, status="error", **msg)
```

Comment:
what is the `status` kwarg for?

Reply:
Cleaned up and broken out to https://github.com/dask/distributed/pull/5926, which this PR incorporates.